public/javascripts/main.js

```diff
@@ -35,12 +35,41 @@ $(function(defaultDateStart, defaultDateEnd){
 
 $(document).ready(function() {
     allSelected();
+
+    checkSelectedMeals();
+
+    $('button.btn-danger').click(function() {
```

Comment:
Based on the message and the selector you would be targeting all `.btn-danger`. Is this for all buttons in all pages, or just for a few?
If it's the latter, please add a more descriptive class name to the button and then update the selector.

Reply:
Yes. This applies to all delete buttons for all pages.